cli/azd/pkg/input/console.go

```diff
@@ -57,6 +58,7 @@ type Console interface {
 	// Prints progress spinner with the given title.
 	// If a previous spinner is running, the title is updated.
 	ShowSpinner(ctx context.Context, title string, format SpinnerUxType)
+	Progress(ctx context.Context, progress string)
```

Comment:
Add description

Reply:
It may also be good to include important details such as "progress should be a single-lined message.

An idea [here](https://github.com/Azure/azure-dev/blob/61901915d9a30b93827281133542c14b8152b631/cli/azd/pkg/project/service_manager.go#L17)